### ISSUE TITLE
running gh-milestone-cleaner version-bumps throws an error: Value was either too large or too small for an Int32.

### ISSUE THREAD

**Issue by nils-a** [nils-org/GitHubMilestoneCleaner#160]
```text
OverflowException: Value was either too large or too small for an Int32.
  at void ThrowInt32OverflowException()
  at int ToInt32(long value)
  at object ChangeType(object value, Type conversionType, IFormatProvider provider)
  at object DeserializeObject(object value, Type type) in SimpleJson.cs:1437
  at object DeserializeObject(object value, Type type) in SimpleJsonSerializer.cs:205
  at object DeserializeObject(object value, Type type) in SimpleJson.cs:1492
  at object DeserializeObject(object value, Type type) in SimpleJsonSerializer.cs:205
  at object DeserializeObject(string json, Type type, IJsonSerializerStrategy
     jsonSerializerStrategy) in SimpleJson.cs:584
  at T DeserializeObject<T>(string json, IJsonSerializerStrategy jsonSerializerStrategy) in
     SimpleJson.cs:596
  at T Deserialize<T>(string json) in SimpleJsonSerializer.cs:22
  at IApiResponse<T> DeserializeResponse<T>(IResponse response) in JsonHttpPipeline.cs:44
  at async Task<IApiResponse<T>> Run<T>(IRequest request, CancellationToken cancellationToken,
     Func<object, object> preprocessResponseBody) in Connection.cs:752
  at async Task<T> Post<T>(Uri uri, object data, string accepts, string contentType,
     CancellationToken cancellationToken)
  at void MoveNext() in GitHubClientExtensions.cs:57
  at async Task<TResult> ImplementationAsync<TResult>(Func<Context, CancellationToken,
     Task<TResult>> action, Context context, CancellationToken cancellationToken,
     ExceptionPredicates shouldRetryExceptionPredicates, ResultPredicates<TResult>
     shouldRetryResultPredicates, Func<DelegateResult<TResult>, TimeSpan, int, Context, Task>
     onRetryAsync, int permittedRetryCount, IEnumerable<TimeSpan> sleepDurationsEnumerable,
     Func<int, DelegateResult<TResult>, Context, TimeSpan> sleepDurationProvider, bool
     continueOnCapturedContext)
  at async Task<TResult> ExecuteAsync<TResult>(Func<Context, CancellationToken, Task<TResult>>
     action, Context context, CancellationToken cancellationToken, bool
     continueOnCapturedContext)
  at async Task<T> WithRetry<T>(GitHubClient client, Func<GitHubClient, Task<T>> operation) in
     GitHubClientExtensions.cs:56
  at async Task RemoveMilestone(Repository repo, Issue issue, string comment) in
     GitHubAdapter.cs:80
  at void MoveNext() in AutoCleanupCommand.cs:130
  at void MoveNext() in AutoCleanupCommand.cs:146
  at void MoveNext() in Progress.cs:103
  at void MoveNext() in Progress.cs:138
  at async Task<T> RunAsync<T>(Func<Task<T>> func) in DefaultExclusivityMode.cs:40
  at async Task<T> StartAsync<T>(Func<ProgressContext, Task<T>> action) in Progress.cs:121
  at async Task StartAsync(Func<ProgressContext, Task> action) in Progress.cs:101
  at async Task<int> ExecuteAsync(CommandContext context, Settings settings) in
     AutoCleanupCommand.cs:141
  at async Task<int> Execute(CommandTree leaf, CommandTree tree, CommandContext context,
     ITypeResolver resolver, IConfiguration configuration) in CommandExecutor.cs:166
```

**Comment by nils-a** [comment]
This is probably an issue with Octokit < v12, so #149 should have solved that problem.
(However, the current released version references Octokit 11.)

**Comment by nils-a** [comment]
Already fixed by #149!
